README.CentOS7.md

```diff
@@ -0,0 +1,154 @@
+# LORIS CentOS 7.x Notes
+
+This document contains supplemental details on how to perform a basic CentOS 7.x install of LORIS.
+Note that the main README in LORIS assumes that LORIS is being run on Ubuntu.
+
+This Readme assumes you already understand basic UNIX, MariaDB (MySQL) and Apache setup and
+settings. If you're not already comfortable troubleshooting system administration issues,
+you should not follow this guide.
```

Comment:
I would replace the last phrase by something more like
These step should be done by someone with Linux command line knowledge.
Currently it is written to scare people away.

Reply:
Great. just made it slightly less scary in [commit 49320e](https://github.com/aces/Loris/pull/5203/commits/49320e31d1884ae975eebed30f66a7e2399622f1) - could you verify/re-review @PapillonMcGill ? - thx

Reply:
@christinerogers Way better, I just requested a little change to current text. (see below).